### PR TITLE
First approach

### PR DESCRIPTION
**Pull request by lyakhovka**
This is the base implementation:

Cezar.java - is the class for the algorithm itself
Secret.java - is for the object created from input data
Direction - is the enum for directions the algorithm can move through the alphabet.
secret file - is just a temporary test file.

### CODE REVIEW

File: Cezar.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+public class Cezar {
+
+    private final String[] alphabet = {"a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"};
+
+    private final Map<String, Integer> lOrder = getStringIntegerMap();
```
Comment (Volodymyr95) [inline Cezar.java]: lOrder what this name is mean?

File: Cezar.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+public class Cezar {
+
+    private final String[] alphabet = {"a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"};
```
Comment (Volodymyr95) [inline Cezar.java]: what about Upper letters?

File: Cezar.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    //
+    // all symbols that are not present in the alphabet stay unchanged.
+    public String render(Path filePath, int key, Direction direction) {
+
+        String inputString = getInputString(filePath);
+        Pattern perSymbol = Pattern.compile("");
+        String[] inputArray = perSymbol.split(inputString);
+
+        if (key > 26) {
+            key = key % 26;
```
Comment (Volodymyr95) [inline Cezar.java]: 26 is magic number

File: Cezar.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        }
+
+
+        switch (direction) {
+            case FORWARD:
+                return moveForward(inputArray, key);
+            case BACKWARD:
+                return moveBackward(inputArray, key);
+            default:
+            { System.out.println("Something went wrong");
```
Comment (Volodymyr95) [inline Cezar.java]: {} - not needed here

File: Cezar.java [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        return res;
+    }
+
+    public String getInputString(Path filePath) {
+        String res = "";
+        try {
+            res = Files.readString(filePath);
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+            throw new RuntimeException(e);
```
Comment (Volodymyr95) [inline Cezar.java]: why u re-throwing exception? do u have some exception handler for it?

File: Cezar.java (hunk 1 above)
Comment (lyakhovka) [reply Cezar.java]: that means 'letters order'

File: Cezar.java (hunk 2 above)
Comment (lyakhovka) [reply Cezar.java]: should I cover upper case too? This is first approach, so upper case is not covered yet.

File: Cezar.java (hunk 3 above)
Comment (lyakhovka) [reply Cezar.java]: Do you mean I should describe this constant in comments? Or should I introduce it as a constant or something?

File: Cezar.java (hunk 4 above)
Comment (lyakhovka) [reply Cezar.java]: got it, will be fixed.

File: Cezar.java (hunk 5 above)
Comment (lyakhovka) [reply Cezar.java]: No, this wrapping was automatically added by Intelij, will fix this.

File: Cezar.java (hunk 3 above)
Comment (lyakhovka) [reply Cezar.java]: Ah, got it. It should be alphabet.length. Will be fixed.

File: Cezar.java (hunk 2 above)
Comment (lyakhovka) [reply Cezar.java]: renamed to 'lettersOrder'

Comment (lyakhovka) [reply Cezar.java]: added support for uppercase

File: Cezar.java (hunk 1 above)
Comment (lyakhovka) [reply Cezar.java]: renamed to lettersOrder

File: Cezar.java (hunk 3 above)
Comment (lyakhovka) [reply Cezar.java]: Fixed

File: Cezar.java (hunk 5 above)
Comment (lyakhovka) [reply Cezar.java]: fixed

File: Cezar.java (hunk 4 above)
Comment (lyakhovka) [reply Cezar.java]: fixed

**Pull request merged**